DeusEx/Classes/DXRKeys.uc

```diff
@@ -146,6 +146,13 @@ function AdjustRestrictions(int doorspickable, int doorsdestructible, int device
     {
         if( d.KeyIDNeeded != 'None' ) l("found door with KeyIDNeeded == " $ d.KeyIDNeeded);
         //if( d.KeyIDNeeded == 'None' || d.bPickable || d.bBreakable ) continue;
+
+        if( d.bHighlight == false ) {
+            d.bPickable = false;
+            d.bLocked = true;
+            d.bHighlight = true;
+            continue;
```

Comment:
basically if a DeusExMover is not highlightable, don't change anything about it gameplay-wise, just show to the player that it is unpickable and indestructible